### PR TITLE
Allowed endilega before the imperative mood; verb frames updated

### PR DESCRIPTION
**Pull request by Holado**

### CODE REVIEW

**Comment by Holado** [comment]
@vthorsteinsson Newest pull request for GreynirCorrect (https://github.com/mideind/GreynirCorrect/pull/48) depends on this one.

File: src/reynir/Greynir.grammar [hunk 1]
```diff
@@ -152,8 +152,10 @@ MgrInnihald →
     Forskeyti? Yfirsetning StLiður* Lokatákn?
     | Staðhæfing
 
+# "Allt er klárt," segir Jón og bætir við: "Komum þessu í gang."
 Staðhæfing →
     Sagt Segjandi Lokatákn?
+    | Sagt Segjandi Segir SegirÁfram Sagt Lokatákn?
```
Comment (vthorsteinsson) [inline src/reynir/Greynir.grammar:158]: Better style (and probably better performance) to have this as one production with an optional part:

```
Staðhæfing →
    Sagt Segjandi ÁframSagt? Lokatákn?

ÁframSagt  →
    Segir SegirÁfram Sagt

```

File: src/reynir/Greynir.grammar [hunk 2]
```diff
@@ -209,6 +211,15 @@ Segir →
...
     "," SögnSegja/tala/pers Heimild_nf/tala/pers AtvFsAuka?
 
+SegirÁfram →
+    "og:st" SögnSegja/tala/pers SegirLíka?
+    | "og:st" SögnSegjaÁfram/tala/pers
+
+$score(+999) SegirÁfram
+
+SegirLíka →
+    "einnig" | "líka"
```
Comment (vthorsteinsson) [inline src/reynir/Greynir.grammar:221]: Add categories here (`:ao` probably)

File: src/reynir/Greynir.grammar [hunk 3]
```diff
@@ -1144,6 +1160,10 @@ SetningSoSagnliður →
     # Virðist sem Björn hafi fengið aðsvif
     | VirðistSem
 
+
+# Atviksorð leyfileg á undan boðháttarsetningu
+BhAtviksliður → "endilega:ao"
```
Comment (vthorsteinsson) [inline src/reynir/Greynir.grammar:1165]: Add "low" "key" ;-)

No, just joking.

File: src/reynir/Greynir.grammar (hunk 1 above)
Comment (Holado) [reply src/reynir/Greynir.grammar:158]: Much sleeker! Updated.

File: src/reynir/Greynir.grammar (hunk 2 above)
Comment (Holado) [reply src/reynir/Greynir.grammar:221]: Done

**Review by vthorsteinsson — COMMENTED** [review]
Good to go!

**Pull request merged**